src/utils/url.ts

```diff
@@ -37,5 +37,9 @@ export const isRemoveJourney = (req: Request): boolean => {
     }
   }
 
+  if (req.url !== undefined && req.url.includes("/remove/")) {
```

Comment:
I would make /remove/ a constant in the constants file. As we already have the REMOVE_SECTION for the url building, you could have a REMOVE_URL_IDENTIFIER = "/" + REMOVE_SECTION and use that in the if check.

Reply:
Updated